pkg/provider/kubernetes/preset.go

```diff
@@ -17,19 +17,14 @@ limitations under the License.
 package kubernetes
 
 import (
-	"bufio"
 	"context"
 	"fmt"
-	"io/ioutil"
-	"os"
-
```

Comment:
Please check the verify job and fix the errors it reported regarding the import statements.

Reply:
> Please also adjust https://github.com/kubermatic/kubermatic/blob/master/pkg/controller/operator/master/resources/kubermatic/api.go#L99

Done 👍